sdk/ml/azure-ai-ml/azure/ai/ml/operations/_workspace_operations_base.py

```diff
@@ -114,6 +115,26 @@ def begin_create(
 
         workspace.resource_group = resource_group
         template, param, resources_being_deployed = self._populate_arm_paramaters(workspace, **kwargs)
+        # check if create with workspace hub request is valid
+        if workspace._kind == PROJECT_WORKSPACE_KIND and workspace.workspace_hub:
```

Comment:
what if it's an empty project workspace without a hub?

Reply:
right, we don't enforce workspace_hub now.